src/pages/workspace/bills/WorkspaceBillsFirstSection.js

```diff
@@ -54,11 +56,25 @@ const WorkspaceBillsFirstSection = (props) => {
                 <Text>
                     {props.translate('workspace.bills.askYourVendorsBeforeEmail')}
                     {props.user.isFromPublicDomain ? (
-                        <TouchableOpacity
-                            onPress={() => Link.openExternalLink('https://community.expensify.com/discussion/7500/how-to-pay-your-company-bills-in-expensify/')}
-                        >
-                            <Text style={[styles.textBlue]}>example.com@expensify.cash</Text>
-                        </TouchableOpacity>
+                        <>
+                            {
+                                Platform.OS === 'android' || Platform.OS === 'ios'
+                                    ? (
+                                        <TouchableNativeFeedback
+                                            onPress={() => Link.openExternalLink('https://community.expensify.com/discussion/7500/how-to-pay-your-company-bills-in-expensify/')}
+                                        >
+                                            <Text style={[styles.textBlue]}>example.com@expensify.cash</Text>
+                                        </TouchableNativeFeedback>
+                                    )
+                                    : (
+                                        <TouchableOpacity
+                                            onPress={() => Link.openExternalLink('https://community.expensify.com/discussion/7500/how-to-pay-your-company-bills-in-expensify/')}
+                                        >
+                                            <Text style={[styles.textBlue]}>example.com@expensify.cash</Text>
+                                        </TouchableOpacity>
+                                    )
```

Comment:
Why are we using different `Touchable` on different platforms? I do not understand your proposal.

Reply:
It was voted for in the slack. There TouchableOpacity doesn't work in Android and iOS as expected. You can read more on the links provided in my proposal.